keras_cv/layers/preprocessing/cut_mix_test.py

```diff
@@ -0,0 +1,16 @@
+import tensorflow as tf
+from keras_cv.layers.preprocessing.random_cut_mix import CutMix
+
+class MixUpTest(tf.test.TestCase):
+    def test_return_shapes(self):
```

Comment:
I think we will need more test coverage for this class.

Reply:
Agreed, I will begin working on that.

Reply:
Done